reference/provided-images.md

```diff
@@ -10,11 +10,13 @@ The official image server has a set of images available. The following table lis
 
 | Name                        | Android Version | Ubuntu Version | Support Status     | Available since |
 |-----------------------------|-----------------|----------------|------------|---------------|
-| `bionic:android11:amd64`      | 11              | 18.04          | supported | 1.10 |
-| `bionic:android11:arm64`      | 11              | 18.04          | supported | 1.10 |
-| `bionic:android10:amd64`      | 10              | 18.04          | supported | 1.0
-| `bionic:android10:arm64`      | 10              | 18.04          | supported | 1.0
-| `bionic:android7:amd64`       | 7               | 18.04          | unsupported as of 1.10 | 1.0 |
-| `bionic:android7:arm64`       | 7               | 18.04          | unsupported as of 1.10 | 1.0 |
+| `bionic:android12:amd64`    | 12              | 18.04          | supported | 1.12 |
+| `bionic:android12:arm64`    | 12              | 18.04          | supported | 1.12 |
+| `bionic:android11:amd64`    | 11              | 18.04          | supported | 1.10 |
+| `bionic:android11:arm64`    | 11              | 18.04          | supported | 1.10 |
+| `bionic:android10:amd64`    | 10              | 18.04          | supported | 1.0
+| `bionic:android10:arm64`    | 10              | 18.04          | supported | 1.0
+| `bionic:android7:amd64`     | 7               | 18.04          | unsupported as of 1.10 | 1.0 |
+| `bionic:android7:arm64`     | 7               | 18.04          | unsupported as of 1.10 | 1.0 |
```

Comment:
We list some of these images in https://anbox-cloud.io/docs/tut/getting-started as well - should we add Android 12 there? Or is it better users start with an older version, just to be sure it works?

Reply:
Decided to keep as is for now.